tests/test-sessions/test-mongodb-sessions/src/test/java/org/eclipse/jetty/nosql/mongodb/PurgeValidSessionTest.java

```diff
@@ -123,6 +123,75 @@ public void testPurgeValidSession() throws Exception
     }
 
 
+    @Test
+    public void testPurgeValidSessionWithPurgeLimitSet() throws Exception
```

Comment:
Ran this test locally and it failed at line 175: there were more sessions in the db than expected. Sure you've got the config right here for the purgeFully() to work on line 154?